### ISSUE TITLE
Scaffolding created through generator doesn't include latest angular bootstrap ui version it still point to ~0.13.4

### ISSUE THREAD

**Issue by BhagyashriK** [Swiip/generator-gulp-angular#1139]
bower.json should contain updated  "angular-ui-bootstrap": "^1.3.3" key and not old "angular-bootstrap": "~0.13.4".
![bootstrap-ui](https://cloud.githubusercontent.com/assets/6581855/15771541/248292e0-2989-11e6-85cb-4c59fa573b9e.png)

**Comment by Swiip** [comment]
The version is quite old yes, PR are welcome

**Comment by elimence** [comment]
Hi, I generated a project a few days before a PR was made to fix this issue, is there a way to update my project to get this fix (I've made folder additions to the structure)? or can I simply remove the old angular bootstrap from `bower.json` and `bower_components` and run `bower install --save` ?

**Comment by BhagyashriK** [comment]
Yes you can remove older angular bootstrap from bower.json and bower_components and run command 
`bower install angular-ui-bootstrap --save`

**Comment by elimence** [comment]
thanks, will do that 👍
